doc/api/v8.md

```diff
@@ -134,3 +134,157 @@ setTimeout(function() { v8.setFlagsFromString('--notrace_gc'); }, 60e3);
 [V8]: https://developers.google.com/v8/
 [here]: https://github.com/thlorenz/v8-flags/blob/master/flags-0.11.md
 [`GetHeapSpaceStatistics`]: https://v8docs.nodesource.com/node-5.0/d5/dda/classv8_1_1_isolate.html#ac673576f24fdc7a33378f8f57e1d13a4
+
+## Serialization API
+
+> Stability: 1 - Experimental
+
+### class: v8.Serializer
+<!--
+added: REPLACEME
+-->
+
+#### new Serializer([options])
+
+* `options` {Object} Set of configurable options to set on the serializer.
+  Can have the following fields:
+  * `getDataCloneError` {Function} A callback that takes a single `message`
+    argument and returns an error object. Defaults to the `Error` constructor.
+  * `writeHostObject` {Function} A callback that takes a single `object`
+    argument and is used to write some kind of host object, if possible. If not,
```

Comment:
> what's a host object?

An object that’s exposed from C++, i.e. by Node or an addon. Most of the options & methods here won’t be very useful to the average user… so I’ve gone ahead and implemented `v8.[de]serialize` shorthands on top of these classes.